src/Pelagos/Bundle/AppBundle/Resources/public/js/dif.js

```diff
@@ -611,11 +611,11 @@ function updateDIF(form)
                 type: "PATCH",
                 datatype: "json",
                 data: formData
-            }).success(function(json, textStatus, jqXHR) {
+            }).done(function(json, textStatus, jqXHR) {
                 if (jqXHR.status === 204) {
                     response.status = "success";
                 }
-            }).error(function (json, text, jqXHR) {
+            }).fail(function (json, text, jqXHR) {
```

Comment:
is there a reason to change error to fail? because the response.message or status attribute is empty when it is used

Reply:
.error is a deprecated according to jquery doc

Reply:
I think the response.status and message is not supported by the .fail() jquery, and we need to use `.fail(function( jqxhr, textStatus, error ) {
    var err = textStatus + ", " + error;
    console.log( "Request Failed: " + err );
});`
just an example. probably you can look into it more.